tests/Functional/OrderUpdateTest.php

```diff
@@ -190,6 +190,24 @@ public function testItAllowsToAddAndRemoveDiscountsForTheOrderItemMultipleTimes(
         );
     }
 
+    public function testItAllowsToAddStoreNotes(): void
+    {
+        $this->makeVariantTrackedWithStockAndPrice('000F_office_grey_jeans-variant-0', 100);
+
+        $order = $this->placeOrderProgramatically(quantity: 5);
+
+        $this->loginAsAdmin();
+        $this->addStoreNotes($order->getId(), 'store notes');
+
+        self::assertResponseStatusCodeSame(302);
+
+        $this->getEntityManager()->clear();
+
+        /** @var EditableOrderInterface $order */
+        $order = $this->getOrderRepository()->findOneBy(['tokenValue' => 'TOKEN']);
+        self::assertSame('store notes', $order->getStoreNotes());
+    }
```

Comment:
What do you think about extracting it to the separate test class? I would do it anyway for order item discounts. Could be done in a separate PR though 🖖

Reply:
Fine for me. I just followed your path :D 

Let's do it in another PR